CODIGO/TileEngine.bas

```diff
@@ -820,32 +825,31 @@ Sub DoPasosFxWithoutPos(ByVal charindex As Integer)
             Dim steps_pan As Long
             steps_vol = ao20audio.ComputeCharFxVolume(.Pos)
             steps_pan = ao20audio.ComputeCharFxPan(.Pos)
+
             If Not .Muerto And EstaPCarea(charindex) And .priv <= charlist(UserCharIndex).priv And charlist(UserCharIndex).Muerto = False Then
                 If .Speeding > 1.3 Then
-                    Call ao20audio.playwav(Pasos(CONST_CABALLO).wav(1), False, steps_vol, steps_pan)
+                    Call ao20audio.PlayWav(Pasos(CONST_CABALLO).wav(1), False, steps_vol, steps_pan)
                     Exit Sub
-
                 End If
            
                 .Pie = Not .Pie
 
                 If .Pie Then
                     If MapData(.Pos.x, .Pos.y).Graphic(1).GrhIndex > 0 Then
                         FileNum = GrhData(MapData(.Pos.x, .Pos.y).Graphic(1).GrhIndex).FileNum
-                        TerrenoDePaso = GetTerrenoDePaso(FileNum)
-                        Call ao20audio.playwav(Pasos(TerrenoDePaso).wav(1), False, steps_vol, steps_pan)
+                        TerrenoDePaso = GetTerrenoDePaso(FileNum, MapData(.Pos.x, .Pos.y).Graphic(2).GrhIndex)
```

Comment:
there is something interesting with the function DoPasosFxWithoutPos... it doesn't seem to be called anywhere, we should remove this

Reply:
deleted